FrontEnd/src/components/ProfileDetail/BannerImage.jsx

```diff
@@ -0,0 +1,32 @@
+import { useMemo } from 'react';
+import classes from './BannerImage.module.css';
+
+function BannerImage ({ data }) {
+    const profile = useMemo(() => {
```

Comment:
Utilizing useMemo here seems redundant. You do not have `memo` wrapper and I do not think that we would benefit from it here. You can use `data. banner_image` directly

Reply:
Done